launcher-implementation/src/main/scala/xsbt/boot/Launch.scala

```diff
@@ -215,6 +219,10 @@ class Launch private[xsbt] (val bootDirectory: File, val lockBoot: Boolean, val
         else
           existing(app, ScalaOrg, explicitScalaVersion, baseDirs(None)) getOrElse retrieve()
 
+      case class TestInterfaceLoader(jar: File, parent: ClassLoader) extends URLClassLoader(Array(jar.toURI.toURL), topLoader)
+      retrievedApp.fullClasspath.find(_.toString.endsWith("test-interface-1.0.jar")) foreach { f =>
```

Comment:
I don't plan on changing the test-interface any time soon, but maybe it would be safer to make this a regex match that's not hardcoded to 1.0.

Reply:
Yeah, I don't like this either. It would be ideal if there was an update report where I could explicitly find the jar corresponding to the test-interface dependency. In the meantime, I'll take your suggestion to convert this to a regex.

Reply:
fixed by https://github.com/sbt/launcher/pull/66/commits/e9698fdff37f9e28f74eaf3f27a04ccfab80793d